CMakeLists.txt

```diff
@@ -121,17 +122,24 @@ IF(NOT DEFINED PTHREADPOOL_SOURCE_DIR)
   SET(PTHREADPOOL_SOURCE_DIR "${CMAKE_BINARY_DIR}/pthreadpool-source" CACHE STRING "pthreadpool source directory")
 ENDIF()
 
-IF(XNNPACK_BUILD_TESTS AND NOT DEFINED GOOGLETEST_SOURCE_DIR)
+IF(XNNPACK_BUILD_TESTS AND NOT DEFINED GOOGLETEST_SOURCE_DIR AND NOT USE_SYSTEM_LIBS)
   MESSAGE(STATUS "Downloading Google Test to ${CMAKE_BINARY_DIR}/googletest-source (define GOOGLETEST_SOURCE_DIR to avoid it)")
   CONFIGURE_FILE(cmake/DownloadGoogleTest.cmake "${CMAKE_BINARY_DIR}/googletest-download/CMakeLists.txt")
   EXECUTE_PROCESS(COMMAND "${CMAKE_COMMAND}" -G "${CMAKE_GENERATOR}" .
     WORKING_DIRECTORY "${CMAKE_BINARY_DIR}/googletest-download")
   EXECUTE_PROCESS(COMMAND "${CMAKE_COMMAND}" --build .
     WORKING_DIRECTORY "${CMAKE_BINARY_DIR}/googletest-download")
   SET(GOOGLETEST_SOURCE_DIR "${CMAKE_BINARY_DIR}/googletest-source" CACHE STRING "Google Test source directory")
+ELSEIF(XNNPACK_BUILD_TESTS AND NOT DEFINED GOOGLETEST_SOURCE_DIR AND USE_SYSTEM_LIBS)
+  SET(SYSTEM_GOOGLETEST_SOURCE_DIR "/usr/src/googletest")
```

Comment:
This is not compatible with the variety of XNNPACK-supported environments (Windows, MSYS, Emscripten).

Also, I don't think package managers need this. They normally don't install tests & benchmarks anyway.

Reply:
OK. I think I'd better keep this downstream. Actually Debian recommends the maintainers to run tests.